src/GeneralStateTestsFiller/stEWASMTests/createRevertFiller.yml

```diff
@@ -0,0 +1,142 @@
+          (memory 1)
+          (data (i32.const 0) "\be\ef")
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            (call $revert
+              (i32.const 0) ;; data offset
+              (i32.const 2)))) ;; data length
+    deadbeef00000000000000000000000000000000:
+      balance: '100000000000'
+      nonce: ''
+      storage: {}
+      code: |
+        (module
+          (import "ethereum" "create" (func $create (param i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+          (import "ethereum" "getExternalCodeSize" (func $getExternalCodeSize (param i32) (result i32)))
+          (import "ethereum" "externalCodeCopy" (func $externalCodeCopy (param i32 i32 i32 i32)))
+          (memory 1)
+          ;; Storage keys
+          (data (i32.const 0)  "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00") ;; 0
+          (data (i32.const 32) "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01") ;; 1
+          (data (i32.const 224) "\a0\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01") ;; external address
+
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            ;; locals
+            (local $codeMemLoc i32)
+            (local $codeLen i32)
+            (local $addrOffset i32)
+            (local $memPointerStorageKey1 i32)
+            (local $memPointerStorageKey2 i32)
+            (local $memPointerCreateValue i32)
+            (local $memPointerCreateResult i32)
+
+            ;; Location and length of code stored in (data) section above
+            (set_local $codeMemLoc (i32.const 256))
+            (set_local $codeLen (i32.const 91))
+            (set_local $addrOffset (i32.const 224))
+
+            ;; memory layout and pointers
+            (set_local $memPointerStorageKey1 (i32.const 0))
+            (set_local $memPointerStorageKey2 (i32.const 32))
+            (set_local $memPointerCreateValue (i32.const 64))
+            (set_local $memPointerCreateResult (i32.const 96))
+
+            ;; get external code
+            (set_local $codeLen 
+              (call $getExternalCodeSize (get_local $addrOffset)))
+
+            (call $externalCodeCopy
+              (get_local $addrOffset)  ;; address
+              (get_local $codeMemLoc)  ;; result
+              (i32.const 0)            ;; code offset
+              (get_local $codeLen))    ;; code length
+
+            
+            ;; initialize
+            ;; call create with value transfer 0x1234
+            (i32.store (get_local $memPointerCreateValue) (i32.const 0x1234))
+
+            ;; call CREATE and store the return success/failure value
+            (call $storageStore
+              (get_local $memPointerStorageKey1)
+              (call $create
```

Comment:
The return value  is 0, 1, or 2. If you want to store in storage you need to store into a memory location first.